packages/dd-trace/src/config.js

```diff
@@ -253,6 +253,11 @@ class Config {
       process.env.DD_TRACE_AGENT_PROTOCOL_VERSION,
       '0.4'
     )
+    const DD_TRACE_PARTIAL_FLUSH_ENABLED = coalesce(
+      options.partialFlushEnabled,
```

Comment:
Also, any input on the name of this config options/the fact that this feature also lives under a config option is also appreciated